test-requirements.txt

```diff
@@ -4,4 +4,4 @@ testrepository
 testtools
 tox
 wheel
-yapf==0.8.0
+yapf==0.9.0
```

Comment:
Just an idea, but as we are always using the latest version of yapf, maybe we do not need to pin it.